@rouault fails on windows with `auto_generated/processing/pdal/qgspdalalgorithms.sip(23): fatal error C1083: Cannot open include file: 'qgspdalalgorithms.h': No such file or directory` so i guess related to my PR but then.. *shrug*

--- a/src/analysis/CMakeLists.txt
+++ b/src/analysis/CMakeLists.txt
@@ -587,13 +587,10 @@ target_include_directories(qgis_analysis PUBLIC
   ${CMAKE_SOURCE_DIR}/src/analysis/vector/
   ${CMAKE_SOURCE_DIR}/src/analysis/vector/geometry_checker
   ${CMAKE_SOURCE_DIR}/external/tinygltf
-
   ${CMAKE_BINARY_DIR}/src/analysis
-
-  if (WITH_PDAL AND NOT PDAL_2_5_OR_HIGHER)
-     ${CMAKE_SOURCE_DIR}/src/analysis/processing/pdal
-  endif()
+  ${CMAKE_SOURCE_DIR}/src/analysis/processing/pdal
 )
+
 
 GENERATE_EXPORT_HEADER(
    qgis_analysis

